### ISSUE TITLE
how about  enhance find file function

### ISSUE THREAD

**Issue by Zenglening** [h3imdall/ftp-remote-edit#109]
“find remote path” need input completed file path and don't support completion. So how about doing a new search box which support 'tab' to completion, like shell.
By the way, this is best sftp plugin I have saw, think you

**Comment by cschindl** [comment]
Hi @Zenglening,

your suggestion sounds interesting. I add it as a feature request.

**Comment by cschindl** [comment]
I've seen you fork the project. Would you like to help us?

**Comment by Zenglening** [comment]
I am not good at node.js， but I have free time now.  So I will have a try.  emmm, I will try my best.

**Comment by cschindl** [comment]
@Zenglening Don't worry about your English skills. English is also not my mother tongue. I think it's great that you want to help us. If you have any questions, feel free to contact me.

**Comment by Zenglening** [comment]
Ok, Can I communicate with you by email? I think it is more convenience  My email is zenglening@qq.com

**Comment by kweensey** [comment]
Agreed, it would be great if it would work just like Fuzzy Finder on local project.. :-)

![image](https://user-images.githubusercontent.com/30492315/32311675-34642702-bf9a-11e7-9d71-d1544c72a5b5.png)

**Comment by cschindl** [comment]
Hi @kweensey,

The Fuzzy Finder search has been implemented and will be available in the new release.

**Comment by kweensey** [comment]
Hello, @cschindl, 

thanks for the update, can't wait to try that out!
